@xen-orchestra/web-core/lib/components/UiCard.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="ui-card" :style="{ flexDirection: direction }">
```

Comment:
If we change the prop to `horizontal`, this can be refactored (don't forget to update the CSS):
```suggestion
  <div class="ui-card" :class="{ horizontal }">
```
Also, when possible, use classes instead of inline CSS.